lib/cc/engine/bundler_audit/analyzer.rb

```diff
@@ -0,0 +1,42 @@
+module CC
+  module Engine
+    module BundlerAudit
+      class Analyzer
+        GemfileLockNotFound = Class.new(StandardError)
+
+        def initialize(directory:, io: STDOUT)
+          @directory = directory
+          @io = io
+        end
+
+        def run
+          if gemfile_lock_exists?
+            Dir.chdir(directory) do
+              Bundler::Audit::Scanner.new.scan do |vulnerability|
+                result = Result.new(vulnerability, File.open(gemfile_lock_path))
+                issue = result.to_issue
```

Comment:
WDYT of changing `to_issue` below to `as_json` and then you could just do `result.to_json` below